src/factories/list/FactoriesKanban.tsx

```diff
@@ -5,17 +5,23 @@ import {
   KanbanBoard,
   moveCard,
 } from '@caldwell619/react-kanban';
-import { Factory, FactoryProgressStatus } from '@/factories/Factory';
+import { FactoryProgressStatus } from '@/factories/Factory';
 import { ProgressChip } from '@/factories/components/ProgressChip';
 import './FactoriesKanban.css';
 import { FactoryGridCard } from '@/factories/list/FactoryGridCard';
 
+type FactoryCard = {
+  id: string;
+  name?: string;
+  progress?: FactoryProgressStatus;
+};
+
 export const FactoriesKanban = () => {
   const gameId = useStore(state => state.games.selected);
 
   const factories = useGameFactories(gameId);
 
-  const board: KanbanBoard<Factory> = {
+  const board: KanbanBoard<FactoryCard> = {
```

Comment:
Shouldn't this be memoized, like `useMemo(() => ...)`? Not sure about react-kanban, but I feel like this could trigger unwanted re-renders

Reply:
I'm not sure. On the other hand, most re-renders are going to be caused by either the factory list being changed, or the filters changed. So most of the re-renders _are_ going to trigger useMemo. I'd rather wait than optimize prematurely, not knowing whether this is actually needed or not.